scripts/run_validation.py

```diff
@@ -111,10 +116,26 @@ def run_validation(args: Validation_args):
     # install dictionaries if needed
     fill_cache_with_dictionaries(shared_cache, args)
     rules = get_rules(args)
-    data_service = DataServiceFactory(config, shared_cache).get_data_service(
-        args.dataset_paths
-    )
-    datasets = data_service.get_datasets()
+    max_dataset_size = get_max_dataset_size(args.dataset_paths)
+    data_service = DataServiceFactory(
+        config, shared_cache, max_dataset_size=max_dataset_size
+    ).get_data_service()
+    large_dataset_validation: bool = data_service.dataset_class != PandasDataset
```

Comment:
`data_service.dataset_class` is always `PandasDataset`, even when` DataServiceFactory.get_dataset_type` is `DaskDataset`.
I think this is because` dataset_class` is not being set in `DataServiceFactory.get_data_service` like it is in `get_dummy_data_service`, so maybe it is taking the default, `PandasDataset`.

Reply:
Good catch, looks like I forgot to update this after resolving merge conflicts